scripts/v120_dynamic.py

```diff
@@ -0,0 +1,16 @@
+from compas.geometry import Point
+from compas_view2 import app
+
+
+viewer = app.App()
+obj = viewer.add(Point(0, 0, 0))
+
+
+@viewer.on(interval=1000)
```

Comment:
How about now @brgcode ? 
There is also option to use timeout argument which just fire once.
I think we can get this merged?

Reply:
we can but it will currently only work on Point objects :)